index.html

```diff
@@ -2,56 +2,107 @@
 <html lang="en">
 
 <head>
-    <meta charset="UTF-8" />
-    <title id="title">
-        <h1>Survey Form</h1>
-    </title>
-    <link rel="stylesheet" href="css/style.css" />
+  <meta charset="utf-8">
+  <title>Home</title>
```

Comment:
Use a more descriptive title to name this HTML document. Example; You can use `<title>Portfolio</title>` if you are naming a portfolio project. The content of the title is one of the components used by search engine algorithms to decide the order in which to list pages in search results.

Reply:
I have made the change, thank you for pointing that out.😊